pom.xml

```diff
@@ -173,6 +186,7 @@
             <detectOfflineLinks>false</detectOfflineLinks>
             <!-- Only show warnings and errors -->
             <quiet>true</quiet>
+            <failOnWarnings>true</failOnWarnings>
```

Comment:
I see this is `failOnWarning` for `maven-compiler-plugin` and `failOnWarnings` for `maven-javadoc-plugin`. Yay Maven.